### PR TITLE
adds option to write aggregated counts to google sheets.

### PR DESCRIPTION
**Pull request by pashap**
Example usage:

`python howmanypeoplearearound/ -a mon0 -v -s 10 -o test.json --aggbymanufacturer --googlecreds people-counter-test-43d3d89f0cbf.json --sheetname 'Test PeopleCounter'`

### CODE REVIEW

File: howmanypeoplearearound/__main__.py [hunk 1]
```diff
@@ -73,27 +95,34 @@ def fileToMacSet(path):
 @click.option('-m', '--manufacturers', default='', help='read list of known manufacturers from file')
 @click.option('--nocorrection', help='do not apply correction', is_flag=True)
 @click.option('--aggbymanufacturer', help='do not store mac addresses in the output file, instead store aggregate count of cellphones by manufacturer.', is_flag=True)
+@click.option('--sheetname', help='name of google sheet to write to.', default='')
+@click.option('--googlecreds', help='if writing to a google sheet, file which contains credentials to write to the sheet.', default='')
```
Comment (correlator) [inline howmanypeoplearearound/__main__.py:99]: Does it make sense to combine both of these flags into a single flag given that we'll never be able to use one without the other?

File: howmanypeoplearearound/gsheets.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    """
+
+    def __init__(self, creds_file, sheet_title, sheet_headers):
+        self.title = sheet_title
+        self.headers = sheet_headers
+
+        credentials = ServiceAccountCredentials.from_json_keyfile_name(creds_file, SCOPE)
+        self.gc = gspread.authorize(credentials)
+
+        self.wks = self.gc.open(self.title).sheet1
```
Comment (correlator) [inline howmanypeoplearearound/gsheets.py]: what's wks and gc?  

can we callthem `google_credentials` and `google_worksheet`?

File: howmanypeoplearearound/gsheets.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        self._check_header()
+
+    def _check_header(self):
```
Comment (correlator) [inline howmanypeoplearearound/gsheets.py]: rename to `write_header_if_first_row_in_sheet`

**Review by correlator — COMMENTED** [review]
Few comments and variables to rename, but this looks pretty great!

File: howmanypeoplearearound/__main__.py (hunk 1 above)
Comment (pashap) [reply howmanypeoplearearound/__main__.py:99]: That's a good question, what do you have in mind? I'm not sure what the single flag would look like

**Comment by correlator** [comment]
LGTM

**Pull request merged**